.github/workflows/reusable-context.yml

```
@@ -0,0 +1,100 @@
+name: Reusable build context
+
+on:  # yamllint disable-line rule:truthy
+  workflow_call:
+    outputs:
+      # Every referenced step MUST always its set output variable,
```

Is this what you meant?
```suggestion
      # Every referenced step MUST always set its output variable,
```